package.json

```diff
@@ -31,15 +31,15 @@
     "prismic-reactjs": "^1.3.4",
     "promise.allsettled": "^1.0.4",
     "react": "16.13.1",
-    "react-native": "0.63.4",
+    "react-native": "npm:react-native-tvos@0.63.4",
```

Comment:
Are there any disadvantages to the non-tvOS clients using this version? If so, is it possible to make this dependency contingent on the target?

Reply:
Not that I know of, as the non-tvOS parts (are purported to!) remain unchanged :)